### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by NetWalker108**
Typo fix and minor changes/elaborations/hyperlinking.

### CODE REVIEW

File: docs/src/faq.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Ordinal Theory FAQ
 What is ordinal theory?
 -----------------------
 
-Ordinal theory is is a protocol for assigning serial numbers to satoshis, the
+Ordinal theory is a protocol for assigning serial numbers to satoshis, the
```
Comment (casey) [inline docs/src/faq.md:7]: This is great, thank you!

File: docs/src/faq.md [hunk 2]
```diff
@@ -146,9 +146,9 @@ given Ethereum NFT is stored.
 
 *Inscriptions are much simpler.*
 
-Ethereum NFTs depend on the Ethereum network and virtual machine, which are
-highly complex, constantly changing, and which introduce changes via
-backwards-incompatible hard forks.
+Ethereum NFTs significantly depend on the Ethereum Virtual Machine(a runtime environment on the Ethereum network) and ERC-based Smart Contracts(ERC-721, ERC-1155, ETC), which are
```
Comment (casey) [inline docs/src/faq.md:149]: Unless there are typos, I think the original text is good here. I might be wilin' out on twitter but I want to keep the docs here more neutral.

File: docs/src/faq.md [hunk 3]
```diff
@@ -211,7 +211,7 @@ underlying protocol.
 
 ### RGB and Taro assets?
 
-RGB and Taro are both second-layer asset protocols built on Bitcoin. Compared
+<a href= "https://www.rgbfaq.com/faq/what-is-rgb">RGB</a> and <a href = "https://docs.lightning.engineering/the-lightning-network/taro">Taro</a> are both second-layer asset protocols built on Bitcoin. Compared
```
Comment (casey) [inline docs/src/faq.md:214]: These should use markdown-style links: `[]()`

**Review by casey — CHANGES_REQUESTED** [review]

File: docs/src/faq.md (hunk 2 above)
Comment (NetWalker108) [reply docs/src/faq.md:149]: I was aiming more for detailed clarity to better assert the distinction. The information provided doesn't break any form of neutrality or promote a stance of any sort, it simply educates.

**Pull request closed without merging**